app/services/api_cost_tracker.py

```
@@ -0,0 +1,879 @@
+                total_calls = 0
+                successful_calls = 0
+                failed_calls = 0
+                
+                for day in range(1, 32):
+                    try:
+                        day_key = now.replace(day=day).strftime("%Y-%m-%d")
+                        total_cost += float(await redis.get(f"cost:daily:{day_key}") or 0)
+                        total_calls += int(await redis.get(f"calls:daily:{day_key}") or 0)
+                        successful_calls += int(await redis.get(f"success:daily:{day_key}") or 0)
+                        failed_calls += int(await redis.get(f"failure:daily:{day_key}") or 0)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)